extensions/README.md

```diff
@@ -1,4 +1,26 @@
 # How to write extensions for IOTA Swarm node
 
-It is straighforard to implement extensions for IOTA Swarm nodes, allowing
-operations being taken regardless of fullnodes.
+This is a guide for how to write a swarm node extension.
```

Comment:
Send another pull request to improve `extensions/README.md` with the aid of Spin.